### PR TITLE
Add filterrific

### PR DESCRIPTION
**Pull request by moisesnandres**

### CODE REVIEW

File: app/models/order_detail.rb [hunk 1]
```diff
@@ -5,4 +5,17 @@ class OrderDetail < ApplicationRecord
   def default_chasis
     self.chasis = Faker::Code.asin
```
Comment (lshimokawa) [inline app/models/order_detail.rb]: Remove this

File: app/models/order_detail.rb [hunk 2]
```diff
@@ -5,4 +5,17 @@ class OrderDetail < ApplicationRecord
...
+
+  filterrific(
+    default_filter_params: { sorted_by: 'created_at_desc' },
+    available_filters: %i[
+      sorted_by
+      search_query
+    ]
+  )
+
+  scope :search_query, lambda { |query|
```
Comment (lshimokawa) [inline app/models/order_detail.rb]: Use a single line.

File: app/models/order_detail.rb [hunk 3]
```diff
@@ -5,4 +5,17 @@ class OrderDetail < ApplicationRecord
   def default_chasis
     self.chasis = Faker::Code.asin
   end
+
+  filterrific(
+    default_filter_params: { sorted_by: 'created_at_desc' },
+    available_filters: %i[
```
Comment (lshimokawa) [inline app/models/order_detail.rb]: Use a single line.

File: app/models/order_detail.rb (hunk 2 above)
Comment (moisesnandres) [inline app/models/order_detail.rb]: @lshimokawa I will close this PR, I did the search [here](https://github.com/devacademyla/divemotor-webapp/pull/9)

**Pull request closed without merging**